src/watcher.js

```diff
@@ -301,6 +313,29 @@ class Watcher extends SafeEventEmitter {
 
         this.on('load.vod', () => observe(vodChatWatcher, $('.vod-chat')[0]));
     }
+
+    directoryObserver() {
+        const observe = (watcher, selector) => {
+            // timeout used because events sometimes fire before elements are created
+            setTimeout(() => {
```

Comment:
you should not need a timeout here, if the element does not exist you've likely not selected the wrapping element properly